src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml.cs

```diff
@@ -46,12 +47,16 @@ public sealed partial class MigrationWizardWindow : DialogWindow, IProgress<Migr
 
         private CancellationTokenSource cancellationTokenSource;
 
-        internal MigrationWizardWindow(BoundSonarQubeProject oldBinding, IConnectedModeMigration connectedModeMigration, ILogger logger)
+        internal MigrationWizardWindow(BoundSonarQubeProject oldBinding,
+            IConnectedModeMigration connectedModeMigration,
+            Action onShowHelp,
```

Comment:
I could have passed the `IBrowserService` to the wizard but it doesn't really need to know how to handle the request. Instead, an `Action` will do just as well, and we can put the logic for handling the call in the controller.